packages/web/vite.config.ts

```diff
@@ -77,6 +77,7 @@ export default defineConfig(async ({ mode }) => {
       'process.env': env
     },
     optimizeDeps: {
+      include: ['@audius/fetch-nft'],
```

Comment:
curious about this

Reply:
needed when i was linking the local fetch-nft in my laptop. removing...